plat/kvm/arm/entry64.S

```diff
@@ -36,6 +36,13 @@
 #include <uk/config.h>
 #include <arch/reloc.h>
 
+/* Prefer using in-image stack, to avoid conflicts when unmapping is done by
```

Comment:
I do not think a page is that much in our usual scenarios. The problem I encountered related to this line was 4 months ago, but from what I vaguely remember, it caused problems when setting up the memory regions, because I would have had to manually add a hardcoded separate function call just to insert the related memory region. Furthermore, at the time of doing this change I also did not have the `umap_bpt_mrd` idea, so a lot has changed since then. I will check again what happens if I do not do this, as I do not remember very well my intentions here.

Reply:
@michpappas I just checked, the reason why I did this is because the paging initialization code, that also unmaps everything outside the kernel, uses the earliest boot stack and therefore ends up unmaps our stack and we end up using an unmapped stack while the paging initialization code is running. I will update the comment to be more verbose.